src/main.rs

```diff
@@ -33,10 +36,25 @@ async fn main() {
         .get_info(tonic_lnd::lnrpc::GetInfoRequest {})
         .await
         .expect("failed to get info");
+    let info_clone = info.into_inner().clone();
+    let info_clone_2 = info_clone.clone();
 
-    let pubkey = PublicKey::from_str(&info.into_inner().identity_pubkey).unwrap();
+    let pubkey = PublicKey::from_str(&info_clone.identity_pubkey).unwrap();
     info!("Starting lndk for node: {pubkey}");
 
+    if !info_clone_2.features.contains_key(&ONION_MESSAGES_OPTIONAL) {
```

Comment:
Add an `info` log indicating that we're going to attempt to set this feature bit? Would rather start verbose for easy debugging then cut back later :)